test/jdk/jdk/incubator/vector/Byte128VectorTests.java

```diff
@@ -5310,6 +5318,92 @@ static void BIT_COUNTMaskedByte128VectorTests(IntFunction<byte[]> fa,
 
 
 
+
+    static byte CTZ(byte a) {
+        return (byte)(CTZ_scalar(a));
```

Comment:
Why not directly `return (byte) (a != 0 ? Integer.numberOfTrailingZeros(a) : 8);` in this method?

Reply:
To keep minimal changes in the test generation template, we want to keep one interface which caters to all the primitive types.